lib/mirage.ml

```diff
@@ -1912,6 +1912,8 @@ module Project = struct
     "open Lwt.Infix\n\
      let return = Lwt.return\n\
      let run = OS.Main.run"
+  let packages = [package "mirage"]
```

Comment:
Previously, functoria were hard-coding that `Project.name` would be the name of the library to link with when building config.ml. This is now a bit more abstract:

https://github.com/mirage/functoria/blob/master/app/functoria_app.ml#L648-L650

This was done primary to make testing of functoria easier (so you can be a bit more explicit when building the tests: https://github.com/mirage/functoria/blob/master/tests/lib/test_app.ml#L60) but I think that's a bit saner to do this instead of relying on `Project.name` being the right ocamlfind name.